.github/workflows/test-internal-prs.yml

```diff
@@ -103,6 +103,36 @@ jobs:
       commit: ${{ github.event.pull_request.head.sha }}
       repository: ${{ github.repository }}
 
+  license-checker:
+    runs-on: ubuntu-latest
+    needs: setup-cache
+    steps:
+      - name: Checkout Amplify UI
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          ref: ${{ inputs.commit }}
+          repository: ${{ inputs.repository }}
+          persist-credentials: false
```

Comment:
You can safely remove inputs in `test-internal-prs.yml`, since it's smart enough to infer `ref` if PR is not from a fork.

```suggestion
```

Reply:
I also update the cache key to `github.event.pull_request.head.sha`